scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -36,7 +33,8 @@ public static BuildTargetResult InitPublish(BuildTargetContext c)
 
         [Target(nameof(PrepareTargets.Init),
         nameof(PublishTargets.InitPublish),
-        nameof(PublishTargets.PublishArtifacts))]
+        nameof(PublishTargets.PublishArtifacts),
+        nameof(TriggerDockerHubBuilds))]
```

Comment:
Can you add `PublishTargets.TriggerDockerHubBuilds` just to maintain uniformity?